CRM/Contact/Form/Search.php

```diff
@@ -546,7 +546,7 @@ public function preProcess() {
 
     // assign context to drive the template display, make sure context is valid
     $this->_context = CRM_Utils_Request::retrieve('context', 'Alphanumeric', $this, FALSE, 'search');
-    if (!CRM_Utils_Array::value($this->_context, self::validContext())) {
+    if (!array_key_exists($this->_context, self::validContext())) {
```

Comment:
Theoretically these aren't equal. I don't see any code outside this class that manipulates the $_validContext property directly, and it seems unlikely anyone would do something screwy with it like change it to null, but it's a public property. Seems low risk though.

Reply:
My thoughts too.